config/sync/core.base_field_override.menu_link_content.menu_link_content.title.yml

```diff
@@ -11,7 +11,7 @@ bundle: menu_link_content
 label: 'Menu link title'
 description: 'The text to be used for this link in the menu.'
 required: true
-translatable: true
```

Comment:
This is nothing to do with your intended changes and undoes some of the already complete and deployed language changes.

I'm going to stop commenting on all the unintended changes here - see overall review comment.